src/Engine/CombatParameters.java

```diff
@@ -11,19 +9,21 @@
 	public GameMap map;
 	public boolean isCounter, canCounter;
 	
-	public CombatParameters(Unit pAttacker, Unit pDefender, GameMap pMap, boolean isCounter, boolean canCounter) {
+	public CombatParameters(Unit pAttacker, Unit pDefender, GameMap pMap, boolean isCounter) {
```

Comment:
Do we ever initialize CombatParameters with isCounter = true? It looks like we probably don't need this parameter, since it is always false.